### ISSUE TITLE
Do we need to duplicate issue answers into actual docs?

### ISSUE THREAD

**Issue by shankari** [e-mission/e-mission-docs#24]
I propose that if an issue is answered, then unless it is super general, we do not need to reflect it back in "docs".

Reasons:
- extra work - we don't have a dedicated doc person on staff
- danger of docs bitrotting/becoming obsolete if people forget to update while with an issue you can always see the timestamp of the last update and know which one is correct
- docs will get too verbose and confusing if every single possible issue is in the docs

Thoughts?

**Comment by bz247** [comment]
Yeah, maybe unless many people have the same problem, it should be okay to only answer them in issues.

**Comment by jackcsullivan** [comment]
Agree that we do not need to reflect it in "docs" as long as it is explicit somewhere in the readme that any users having issues should check the issues tab in the repo (both open and closed issues).

**Comment by shankari** [comment]
in response to https://github.com/e-mission/e-mission-docs/issues/24#issuecomment-414749718, what do you think should be the threshold for *many* people?

**Comment by MetroPietro** [comment]
I agree. However when left in Issues, please note the keywords needed for us to search for the solutions to the problems on our own. RTFM is not sufficient when there are several hundred online "manuals" available for trying to solve questions.

**Comment by deepalics0044** [comment]
Yes I agree. I think reflecting on docs can be haphazard, we can look into the issues if some information is required.

**Comment by jackcsullivan** [comment]
In response to https://github.com/e-mission/e-mission-docs/issues/24#issuecomment-414750041, 5 seems like a reasonable number.

**Comment by shankari** [comment]
@jackcsullivan hm and maybe we can have people vote on confusing issues. Hm, it looks like voting is a frequently requested feature of github but doesn't exist yet.
https://github.com/isaacs/github/issues/9

Any thoughts on how to simulate voting?

**Comment by jackcsullivan** [comment]
@shankari we could all comment numbers and people could thumbs up the ones they like.

**Comment by shankari** [comment]
@jackcsullivan I was talking about voting for issues to highlight that "it was confusing for me and needs to go into the docs"

**Comment by shankari** [comment]
@bz247 that sounds like a great idea.
Let's see if others have any comments in the next couple of days, and then we can make the change.
Where should we put the instructions?
docs repository? `README`?

Also, who wants to do this?

**Comment by jackcsullivan** [comment]
@shankari Have an issues section under "Contributing" in the main e-mission-docs readme that has instructions for the protocol?
